pkg/serverless/flush/strategy_test.go

```diff
@@ -0,0 +1,24 @@
+package flush
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
```

Comment:
I would add some tests for the other strategies as well

Reply:
Very true, adding some.